.travis.yml

```diff
@@ -8,4 +8,8 @@ rvm:
   - 2.3.0
   - jruby
 
+script:
```

Comment:
if it's part of travis it should be part of rake too ... something like https://github.com/abonas/kubeclient/pull/166